prow/plugins/approve/approvers/approvers_test.go

```diff
@@ -148,70 +152,71 @@ func TestGetFiles(t *testing.T) {
 			testName:          "Single Root File PR Approved",
 			filenames:         []string{"kubernetes.go"},
 			currentlyApproved: sets.NewString(rootApprovers.List()[0]),
-			expectedFiles:     []File{ApprovedFile{"", sets.NewString(rootApprovers.List()[0]), "org", "repo", "master"}},
+			expectedFiles: []File{ApprovedFile{
+				githubLink, "", sets.NewString(rootApprovers.List()[0]), "org", "repo", "master"}},
```

Comment:
Let's not change line breaks?

Reply:
No problem, I'll remove the new line break.